DigitalLearningSolutions.Web.Tests/Controllers/LearningPortal/CurrentTests.cs

```diff
@@ -165,30 +165,38 @@ public void Trying_to_edit_complete_by_date_for_non_existent_course_should_retur
         }
 
         [Test]
-        public void Setting_a_valid_complete_by_date_should_call_the_course_service()
+        public void Setting_a_valid_complete_by_date_for_course_should_call_the_course_service()
         {
             // Given
+            const int id = 1;
+            const int progressId = 1;
             const int newDay = 29;
             const int newMonth = 7;
             const int newYear = 3020;
+            var formData = new EditCompleteByDateFormData { Day = newDay, Month = newMonth, Year = newYear };
+
             var newDate = new DateTime(newYear, newMonth, newDay);
-            const int progressId = 1;
 
             // When
-            controller.SetCurrentCourseCompleteByDate(1, newDay, newMonth, newYear, 1);
+            controller.SetCurrentCourseCompleteByDate(id, progressId, formData);
 
             // Then
             A.CallTo(() => courseDataService.SetCompleteByDate(progressId, CandidateId, newDate)).MustHaveHappened();
         }
 
         [Test]
-        public void Setting_an_empty_complete_by_date_should_call_the_course_service_with_null()
+        public void Setting_an_empty_complete_by_date_for_course_should_call_the_course_service_with_null()
         {
             // Given
+            const int id = 1;
             const int progressId = 1;
+            int? day = null;
+            int? month = null;
+            int? year = null;
+            var formData = new EditCompleteByDateFormData { Day = day, Month = month, Year = year };

```

Comment:
Very minor, but there are a couple of tests where these nullable day/month/year variables are defined, put into the formData, and then not used afterwards. Given that the formData construction specifies "Day = day" etc., we could just replace the day/month/year with the nulls in there as it is clear what is going on ("Day = null" etc.).

Reply:
Ah good idea, fixed those 3 tests